libbeat/common/event_test.go

```diff
@@ -22,14 +22,13 @@ import (
 	"testing"
 	"time"
 
-	"github.com/gofrs/uuid"
-	"github.com/stretchr/testify/assert"
-
 	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/elastic/elastic-agent-libs/mapstr"
+	"github.com/gofrs/uuid"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestConvertNestedMapStr(t *testing.T) {
+func BenchmarkTestConvertNestedMapStr(b *testing.B) {
```

Comment:
Please don't convert regular tests to benchmarks. Our CI does not run benchmarks therefore does not fail on their failure.

We need a separate simple benchmark.